plugins/k8sauditeks/rules/k8s_audit_rules.yaml

```diff
@@ -0,0 +1,743 @@
+#
```

Comment:
This file is a replica of the k8saudit one and I'm against having two copies, see comment above.